internal/util/paramtable/component_param.go

```diff
@@ -1674,9 +1674,9 @@ func (p *dataCoordConfig) initSingleCompactionDeltalogMaxNum() {
 	p.SingleCompactionDeltalogMaxNum = p.Base.ParseInt64WithDefault("dataCoord.compaction.single.deltalog.maxnum", 200)
 }
 
-// interval we check and trigger global compaction
+// interval we check and trigger global compaction unit in seconds
 func (p *dataCoordConfig) initGlobalCompactionInterval() {
-	p.GlobalCompactionInterval = time.Duration(p.Base.ParseInt64WithDefault("dataCoord.compaction.global.interval", int64(60*time.Second)))
+	p.GlobalCompactionInterval = time.Second * time.Duration(p.Base.ParseInt64WithDefault("dataCoord.compaction.global.interval", 60))
```

Comment:
shall we sync default value to 10 seconds (from master)

Reply:
![image](https://github.com/milvus-io/milvus/assets/51370125/f8797c4b-2ba3-43d7-8cba-fbfc09255f7c)
Its 60s in master